bundler.go

```diff
@@ -421,9 +421,19 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		}
 	}
 
-	if b.buildTags != "" {
-		tags := []string{"-tags", b.buildTags}
-		cmd.Args = append(cmd.Args[:2], append(tags, cmd.Args[2:]...)...)
+	flags := make([]string, 0)
```

Comment:
I'd rather you build all args of the cmd (including `ldflags` and so on) before running `exec.Command`. Therefore you would have to add this logic at line 406 with something along the line of:

```go
// Build args
args := []string{b.pathGoBinary, "build", "-ldflags", std.String(), "-o", binaryPath}

// TODO Add custom tags here

args = append(args, b.pathBuild)

// Build cmd
...
```